pkg/apis/core/types.go

```diff
@@ -1434,11 +1434,13 @@ type KeyToPath struct {
 	Mode *int32
 }
 
-// Local represents directly-attached storage with node affinity
+// Local represents directly-attached storage with node affinity (Beta feature)
 type LocalVolumeSource struct {
-	// The full path to the volume on the node
-	// For alpha, this path must be a directory
-	// Once block as a source is supported, then this path can point to a block device
+	// The full path to the volume on the node.
+	// It can be either a directory or block device (disk, partition, ...).
+	// Directories can be represented only by PersistentVolume with VolumeMode=Filesystem.
+	// Block devices can use VolumeMode=Block only. And the volumeMode field requires
```

Comment:
This slightly implies that the block alpha feature has to be enabled even for volumeMode= filesystem.  Maybe change this to "Block devices can be represented only by VolumeMode=Block, which also requires the BlockVolume alpha feature gate to be enabled"